client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPluginValidator.js

```diff
@@ -311,3 +313,39 @@ function shallowEquals(a, b) {
 function validCloudUrl(url) {
   return /^(https:\/\/|)[a-z\d-]+\.[a-z]+-\d+\.zeebe\.camunda\.io(:443|)\/?/.test(url);
 }
+
+const checkNumberLimits = (string, min, max) => {
+  const number = parseInt(string);
+  const ret = !isNaN(number) && number >= min && number <= max;
+  return ret;
+};
+
+const validateUrl = (value, message) => {
+  const urlRegex = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}\.[a-z]{2,4}\b([-a-zA-Z0-9@:%_+.~#?&//=]*)/;
+  return urlRegex.test(value) ? null : message;
+};
```

Comment:
Suggestion: Use built-in `new URL`. You will probably need to prepend `http://` if it's missing but we should accept input without the protocol as well.

Reply:
You're right. I guess we don't need to be super strict here anyway